portfolio/src/main/java/com/google/sps/servlets/PokedexDataServlet.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/pokedex-data")
+public class PokedexDataServlet extends HttpServlet {
+
+  private int[] getGenerationCount() {
+    int[] result = new int[10];
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream("/WEB-INF/pokemon.csv"));
+    String header = scanner.nextLine();
```

Comment:
instead of assigning it to a var you wont use, prob better to just call nextLine() with an explanatory comment